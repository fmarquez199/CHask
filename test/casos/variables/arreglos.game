Rune|}01234567890{| = a1
Runes|}0{| a2 = |}~Natascha~, ~Francisco~, ~Manuel~{|

Skill|}1{| indices = |}3'67, 3'20, 3'0{|
Rune|}2{| indices = |}*a*,*b*,*c*{|
Power|}3{| indices = |}0,1,2,3,4,5,6,7,8,9,0{|
Battle|}2{| indices = |}Lose,Win{|

#indices
