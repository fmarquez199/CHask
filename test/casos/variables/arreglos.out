Token Rune en la fila: 1, columna: 1
Token |} en la fila: 1, columna: 5
Token entero 01234567890 en la fila: 1, columna: 7
Token {| en la fila: 1, columna: 18
Token = en la fila: 1, columna: 21
Token identificador "a1" en la fila: 1, columna: 23
Token End Instruction \n en la fila: 1, columna: 25
Token Runes en la fila: 2, columna: 1
Token |} en la fila: 2, columna: 6
Token entero 0 en la fila: 2, columna: 8
Token {| en la fila: 2, columna: 9
Token identificador "a2" en la fila: 2, columna: 12
Token = en la fila: 2, columna: 15
Token |} en la fila: 2, columna: 17
Token string ~Natascha~ en la fila: 2, columna: 19
Token , en la fila: 2, columna: 29
Token string ~Francisco~ en la fila: 2, columna: 31
Token , en la fila: 2, columna: 42
Token string ~Manuel~ en la fila: 2, columna: 44
Token {| en la fila: 2, columna: 52
Token End Instruction \n en la fila: 2, columna: 54
Token End Instruction \n en la fila: 3, columna: 1
Token Skill en la fila: 4, columna: 1
Token |} en la fila: 4, columna: 6
Token entero 1 en la fila: 4, columna: 8
Token {| en la fila: 4, columna: 9
Token identificador "indices" en la fila: 4, columna: 12
Token = en la fila: 4, columna: 20
Token |} en la fila: 4, columna: 22
Token flotante 3'67 en la fila: 4, columna: 24
Token , en la fila: 4, columna: 28
Token flotante 3'20 en la fila: 4, columna: 30
Token , en la fila: 4, columna: 34
Token flotante 3'0 en la fila: 4, columna: 36
Token {| en la fila: 4, columna: 39
Token End Instruction \n en la fila: 4, columna: 41
Token Rune en la fila: 5, columna: 1
Token |} en la fila: 5, columna: 5
Token entero 2 en la fila: 5, columna: 7
Token {| en la fila: 5, columna: 8
Token identificador "indices" en la fila: 5, columna: 11
Token = en la fila: 5, columna: 19
Token |} en la fila: 5, columna: 21
Token caracter *a* en la fila: 5, columna: 23
Token , en la fila: 5, columna: 26
Token caracter *b* en la fila: 5, columna: 27
Token , en la fila: 5, columna: 30
Token caracter *c* en la fila: 5, columna: 31
Token {| en la fila: 5, columna: 34
Token End Instruction \n en la fila: 5, columna: 36
Token Power en la fila: 6, columna: 1
Token |} en la fila: 6, columna: 6
Token entero 3 en la fila: 6, columna: 8
Token {| en la fila: 6, columna: 9
Token identificador "indices" en la fila: 6, columna: 12
Token = en la fila: 6, columna: 20
Token |} en la fila: 6, columna: 22
Token entero 0 en la fila: 6, columna: 24
Token , en la fila: 6, columna: 25
Token entero 1 en la fila: 6, columna: 26
Token , en la fila: 6, columna: 27
Token entero 2 en la fila: 6, columna: 28
Token , en la fila: 6, columna: 29
Token entero 3 en la fila: 6, columna: 30
Token , en la fila: 6, columna: 31
Token entero 4 en la fila: 6, columna: 32
Token , en la fila: 6, columna: 33
Token entero 5 en la fila: 6, columna: 34
Token , en la fila: 6, columna: 35
Token entero 6 en la fila: 6, columna: 36
Token , en la fila: 6, columna: 37
Token entero 7 en la fila: 6, columna: 38
Token , en la fila: 6, columna: 39
Token entero 8 en la fila: 6, columna: 40
Token , en la fila: 6, columna: 41
Token entero 9 en la fila: 6, columna: 42
Token , en la fila: 6, columna: 43
Token entero 0 en la fila: 6, columna: 44
Token {| en la fila: 6, columna: 45
Token End Instruction \n en la fila: 6, columna: 47
Token Battle en la fila: 7, columna: 1
Token |} en la fila: 7, columna: 7
Token entero 2 en la fila: 7, columna: 9
Token {| en la fila: 7, columna: 10
Token identificador "indices" en la fila: 7, columna: 13
Token = en la fila: 7, columna: 21
Token |} en la fila: 7, columna: 23
Token Lose en la fila: 7, columna: 25
Token , en la fila: 7, columna: 29
Token Win en la fila: 7, columna: 30
Token {| en la fila: 7, columna: 33
Token End Instruction \n en la fila: 7, columna: 35
Token End Instruction \n en la fila: 8, columna: 1
Token # en la fila: 9, columna: 1
Token identificador "indices" en la fila: 9, columna: 2
Token End Instruction \n en la fila: 9, columna: 9
